README.md

```diff
@@ -147,6 +147,41 @@ For example, you can ingest databases with a similar name using the [for-cycle](
 
 To make the YAML config file even more DRY, checkout the Jinja2 [variables](http://jinja.pocoo.org/docs/dev/templates/#assignments) and [blocks](http://jinja.pocoo.org/docs/dev/templates/#assignments) and [many other features](http://jinja.pocoo.org/docs/dev/templates/#) that Jinja2 provides.
 
+### wrap the output commands
+
+You can set prefix and postfix values that would wrap the command and its arguments. By default the prefix will be `sqoop import` and the postfix empty
+
+The following (bash) example will retry the sqoop import if it returns a non-zero exitcode 
+
+```
+global:
+  script:
+    prefix: "n=0\nuntil [ $n -ge 5 ]\ndo \n  "
```

Comment:
Maybe this can be a multi-line string, what do you think?
https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines#21699210